### DISCUSSION TITLE
Publish on Flathub

### DISCUSSION THREAD

**Discussion by sojusnik** [facelessuser/Rummage#451] [Ideas]
If possible, it would be very convenient to have this app published as a flatpak on https://flathub.org/ so that it could be easily installed and updated.

**Comment by facelessuser** [comment]
While I can appreciate people's desire for such things, I just don't have the time to dig in and figure out how to host on flathub or anyone else's favorite distribution platform. If someone would like to go through the effort to do and maintain it, I'm sure others would be grateful.
